### PR TITLE
Update list of available plugins

### PR DESCRIPTION
**Pull request by alexzautke**

### CODE REVIEW

File: setting_up_firely_server/configuration/plugins/available_plugins.rst [hunk 1]
```diff
@@ -44,6 +40,13 @@ Infrastructural plugins
 :Order: 130
 :Description: Registers an implementation for the ``ISerializationService`` and ``ISerializationSupport`` interfaces and actual serializers and parsers for JSON and XML.
 
+.. _vonk_plugins_core_services:
+:Name: Terminology Services
+:Configuration: `onk.Plugins.Terminology.TerminologyConfiguration`
```
Comment (cknaap) [inline setting_up_firely_server/configuration/plugins/available_plugins.rst:45]:
```suggestion
:Configuration: `Vonk.Plugins.Terminology.TerminologyConfiguration`
```

File: setting_up_firely_server/configuration/plugins/available_plugins.rst [hunk 2]
```diff
@@ -44,6 +40,13 @@ Infrastructural plugins
 :Order: 130
 :Description: Registers an implementation for the ``ISerializationService`` and ``ISerializationSupport`` interfaces and actual serializers and parsers for JSON and XML.
 
+.. _vonk_plugins_core_services:
```
Comment (cknaap) [inline setting_up_firely_server/configuration/plugins/available_plugins.rst:43]: Should this not be listed as part of the Terminology Service paragraph?

File: setting_up_firely_server/configuration/plugins/available_plugins.rst [hunk 3]
```diff
@@ -44,6 +40,13 @@ Infrastructural plugins
...
+:Name: Terminology Services
+:Configuration: `onk.Plugins.Terminology.TerminologyConfiguration`
+:License token: http://fire.ly/vonk/plugins/terminology
+:Order: 132
+:Description: Registers local and remote terminology services to make them available as a core infrastructur.
```
Comment (cknaap) [inline setting_up_firely_server/configuration/plugins/available_plugins.rst:48]:
```suggestion
:Description: Registers local and remote terminology services to make them available as a core infrastructure.
```

File: setting_up_firely_server/configuration/plugins/available_plugins.rst
Comment (cknaap) [inline setting_up_firely_server/configuration/plugins/available_plugins.rst:1]: I could not fix Terminology - the plugin order numbers from the regular and 'Compatibility' configurations seem mingled.

Comment (cknaap) [inline setting_up_firely_server/configuration/plugins/available_plugins.rst:1]: I missed quite a few items still, and add them in a subsequent commit.

**Review by cknaap — CHANGES_REQUESTED** [review]

File: setting_up_firely_server/configuration/plugins/available_plugins.rst (hunk 2 above)
Comment (alexzautke) [reply setting_up_firely_server/configuration/plugins/available_plugins.rst:43]: I would rather want to keep it here. For me this configuration is all about core infrastructure.

File: setting_up_firely_server/configuration/plugins/available_plugins.rst
Comment (cknaap) [reply setting_up_firely_server/configuration/plugins/available_plugins.rst:1]: The obsolete set of plugin numbers is removed from Vonk altogether in https://github.com/FirelyTeam/Vonk/pull/1714

**Review by cknaap — APPROVED** [review]

**Pull request merged**